cli/mobycli/exec.go

```diff
@@ -136,5 +137,9 @@ func ExecSilent(ctx context.Context, args ...string) ([]byte, error) {
 		args = os.Args[1:]
 	}
 	cmd := exec.CommandContext(ctx, ComDockerCli, args...)
-	return cmd.CombinedOutput()
+	var b bytes.Buffer
```

Comment:
Can't we just replace `cmd.CombinedOutput()` by `cmd.CombinedOutput()` here to do this ? 
In terms of impact, it's OK as version is the only user of `ExecSilent()`, OK